### PR TITLE
Fix rabbitmq builds after move to Cloudsmith mirror

### PR DESCRIPTION
**Pull request by bbezak**
Rabbitmq switched to Cloudsmith repos [1] [2].  Upstream moved as well [3].
Sed which alters repositories during kolla build was not taking into account two lined baseurl.

[1] https://www.rabbitmq.com/blog/2024/08/11/package-repository-updates#packagecloud-will-be-discontinued-on-aug-18th-2024
[2] https://www.rabbitmq.com/blog/2024/08/11/package-repository-updates#mirrors-now-use-rabbitmqcom-domains
[3] https://review.opendev.org/c/openstack/kolla/+/929302

Depends-On: https://github.com/stackhpc/stackhpc-release-train/pull/338

### CODE REVIEW

File: etc/kayobe/stackhpc.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ stackhpc_repo_rabbitmq_erlang_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/
 stackhpc_repo_rabbitmq_erlang_version: "{{ stackhpc_repo_distribution }}"
 
 # RabbitMQ for Redhat family, version 8/9 agnostic
-stackhpc_repo_rabbitmq_server_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rabbitmq/rabbitmq-server/el/8/x86_64/{{ stackhpc_repo_rabbitmq_server_version }}"
+stackhpc_repo_rabbitmq_server_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rabbitmq/rabbitmq-server/el/8/noarch/{{ stackhpc_repo_rabbitmq_server_version }}"
```
Comment (mnasiadka) [inline etc/kayobe/stackhpc.yml]: maybe I'm a bit picky, but why not el9?

File: etc/kayobe/kolla-image-tags.yml [hunk 2]
```diff
@@ -44,5 +44,5 @@ kolla_image_tags:
   ovn:
     rocky-9: 2023.1-rocky-9-20240809T102431
   rabbitmq:
-    rocky-9: 2023.1-rocky-9-20240823T101942
+    rocky-9: 2023.1-rocky-9-20240925T133913
```
Comment (Alex-Welsh) [inline etc/kayobe/kolla-image-tags.yml:51]: New tags need a release note, and we should probably be quite careful when promoting new container images for RabbitMQ. You could try kicking off a Multinode using CI with this branch to test?

Comment (bbezak) [reply etc/kayobe/kolla-image-tags.yml:51]: versions didn't change for main antelope version 3.11, and 3.12. only 3.12 changed:

```
new repo:
rabbitmq-server-3.11.28-1.el8.noarch.rpm
rabbitmq-server-3.12.14-1.el8.noarch.rpm
rabbitmq-server-3.13.7-1.el8.noarch.rpm

old repo:
rabbitmq-server-3.11.28-1.el8.noarch.rpm
rabbitmq-server-3.12.14-1.el8.noarch.rpm
rabbitmq-server-3.13.4-1.el8.noarch.rpm
```

File: etc/kayobe/stackhpc.yml (hunk 1 above)
Comment (bbezak) [reply etc/kayobe/stackhpc.yml]: done

**Review by mnasiadka — DISMISSED** [review]

**Review by Alex-Welsh — APPROVED** [review]

**Pull request merged**